tailwind.config.ts

```diff
@@ -6,7 +6,20 @@ const config: Config = {
     './components/**/*.{js,ts,jsx,tsx,mdx}',
     './app/**/*.{js,ts,jsx,tsx,mdx}',
   ],
+  // darkMode: 'selector', // enable this for manually toggling dark mode
   theme: {
+    screens: {
+      // defaults from https://tailwindcss.com/docs/theme
+      sm: '431px', // 430px is the width of the iPhone 14 Pro Max
```

Comment:
Know I commented about using this size for our mobile breakpoint before - I actually believe tailwind's default `sm` breakpoint of `640px` encourages mobile styling to extend all the way to that screen width, where they expect tablet sizing to begin.

Generally I think it's best to use the default screen size breakpoints so that user experience of our site is consistent with other tailwind sites. 

I recommend removing this custom breakpoint unless we have a good reason to have non-mobile styling kick in at exactly 431px.

Reply:
I can get behind this. @ChTiSh, @kinskiwu What do you think about this suggestion.